.github/workflows/build-test.yml

```diff
@@ -1132,16 +1132,3 @@ jobs:
         run: |
           echo ${{steps.version.outputs.version}}
         shell: bash
-
-      - name: Create GH Release
```

Comment:
for security reasons, we removed the token and this fails. We can do this step manually

Reply:
@joyceerhl  didn't you restore permissions for a few actions recently? Can we do that here?

Reply:
Yep, you would add the following near the top of this file:
```
permissions:
  pull-requests: write
```